pkg/kubelet/kubelet_pods.go

```diff
@@ -1407,6 +1408,11 @@ func (kl *Kubelet) convertStatusToAPIStatus(pod *v1.Pod, podStatus *kubecontaine
 		true,
 	)
 
+	for _, c := range pod.Status.Conditions {
```

Comment:
Better add a comment for this.

Like, "preserve non-system condition types". WDYT?

Reply:
sure